### PR TITLE
Whitelist approach with devise

### PR DESCRIPTION
**Pull request by ManyBaby**

### CODE REVIEW

File: app/controllers/static_pages_controller.rb [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+  skip_before_action :authenticate_user!, %i[home about]
+  before_action :get_editable_fields, only: %i[home about]
   include Cart
 
   def about
     @categories = Category.all
     @editables = EditableFieldsCollection.new({
-      value: @editable_fields
-    })
+                                                value: @editable_fields
```
Comment (DavidGeismarLtd) [inline app/controllers/static_pages_controller.rb:9]: weird indentation

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
 
 private
 
   def set_user
     @user = current_user
+    address
+  end
+
+  def address
     @address = @user.address
```
Comment (DavidGeismarLtd) [inline app/controllers/users_controller.rb:14]:
```suggestion
    @address = @user.address || Address.new
```

**Review by DavidGeismarLtd — APPROVED** [review]
Approved but made some comments

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -1,15 +1,17 @@
...
 
   def set_user
     @user = current_user
-    @address = @user.address
+    address
+  end
+
+  def address
+    @address = @user.address || Address.new
     @address ||= Address.new
```
Comment (DavidGeismarLtd) [inline app/controllers/users_controller.rb:15]: remove that line

Comment (ManyBaby) [inline app/controllers/users_controller.rb:15]:
```suggestion

```

**Pull request merged**